.travis.yml

```diff
@@ -12,11 +12,17 @@ jobs:
   include:
   - env: CMD="make validate"
   - env: CMD="make deploy deploytool=helm globalnet=true"
+  - env: CMD="make deploy deploytool=helm cable_driver=wireguard globalnet=true"
```

Comment:
I wouldn't add this one, there's no value in duplicating the existing jobs each time a new flag is added, otherwise we'll end up with 10's of jobs that do pretty much the same thing.

In fact since the only thing that matters is the flag, i'd add a simple job that tests only that - `make deploy cable_driver=wireguard`

Reply:
I completely understand the problem of the matrix test cases. 
Actually, originally I planed to leave the decision about the automatically running test jobs to the Submariner team. 
Currently with Wireguard we have 2 jobs that cover the following 4 use cases: deployment with operator/helm and with and without `globalnet`. (helm and globalnet are new features.)

Reply:
Yeah same way we only test globalnet in one variation here.
And already theres another flag being added for lighthouse so i dont want to end up with 8 jobs :)